README.md

```diff
@@ -71,10 +71,18 @@ Using the Go Backend SDK, it looks like so:
 ```go
 package hello
 
-//encore:api public
-func Ping(ctx context.Context, params *PingParams) (*PingResponse, error) {
-    msg := fmt.Sprintf("Hello, %s!", params.Name)
-    return &PingResponse{Message: msg}, nil
+import (
+	"context"
+)
```

Comment:
I'd say remove the import statement and use `fmt.Sprintf` below so it's more idiomatic Go